main.go

```diff
@@ -35,9 +35,13 @@ func main() {
 
 	// get new conch and setup cleanup
 	c := newConch(fig)
+	hasClosedChannel := false
 
 	sm.Set(func(s *sigmon.SignalMonitor) {
-		close(c.doneChan())
+		if !hasClosedChannel {
```

Comment:
Not sure I understand. I suppose we could move this into `conch` as in `c.closeDoneChan()`
conch would then ensure that the channel only be closed once.

(I added this because the code allowed me to `Ctrl+c` twice. The first time closes the channel successfully, the second time throws an error regarding closing an already closed channel, as opposed to accepting the second `break`, and terminating in a controlled manner. Conceivably, the OS could send many break requests at once, although it would be rare.)